src/routes/(static)/nos-partenaires/+page.ts

```diff
@@ -0,0 +1,8 @@
+import type { PageLoad } from "../../../../.svelte-kit/types/src/routes";
```

Comment:
```suggestion
import type { PageLoad } from "./$types";
```

Non, il y en a qquns qui se sont faufilés dans le code, mais c'est une erreur de refacto d'une IDE ou d'une autre.
Je corrige les autres sur main